homeassistant/components/rki_covid/config_flow.py

```diff
@@ -0,0 +1,55 @@
+"""Make the component configurable via the UI."""
+from __future__ import annotations
+import logging
+from typing import Any
+import voluptuous as vol
+from homeassistant import config_entries
+from homeassistant.components.rki_covid.coordinator import RkiCovidDataUpdateCoordinator
+
+
+from .const import ATTR_COUNTY, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RKICovidNumbersConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """RKI Covid numbers config flow."""
+
+    VERSION = 1
+
+    _options: dict[str, Any] | None = None
+
+    async def async_step_user(self, user_input: dict[str, Any] | None = None):
+        """Invoke when a user initiates a flow via the user interface."""
+        _LOGGER.debug(f"User triggered configuration flow: {user_input}")
+
+        errors: dict[str, str] = {}
+
+        if self._options is None:
+            self._options = {}
+
+            # add items from coordinator
+            coordinator = RkiCovidDataUpdateCoordinator(self.hass)
+            await coordinator.async_config_entry_first_refresh()
```

Comment:
Lets not use the coordinator here, instead, use the client library.

Reply:
The whole accumulation of the data is done in the coordinator since it has nothing to do with the client library. Wouldn't it be good to keep the coordinator to decrease code duplitaces?

Reply:
> Wouldn't it be good to keep the coordinator to decrease code duplitaces?

It isn't needed to use the coordinator for that, instead, abstract the accumulation in a separate method that is used by the coordinator and the config flow.

Reply:
Thats true, I extracted it and use the parser directly for the config_flow